src/components/code/_code_block.tsx

```diff
@@ -64,15 +105,17 @@ export class EuiCodeBlockImpl extends Component {
     }
 
     if (this.props.language) {
-      hljs.highlightBlock(this.code);
+      if (this.code) {
+        hljs.highlightBlock(this.code);
+      }
 
       if (this.codeFullScreen) {
         hljs.highlightBlock(this.codeFullScreen);
       }
     }
   };
 
-  onKeyDown = event => {
+  onKeyDown = (event: KeyboardEvent<HTMLElement>) => {
```

Comment:
```suggestion
  onKeyDown = (event: KeyboardEvent<HTMLCodeElement>) => {
```

`HTMLCodeElement` is more specific (as `onKeyDown` is always passed to a `code` element)

Reply:
ahh cool I missed that - thanks!  I was also surprised that there isn't a `HTMLCodeElement` type when I was writing this.

Actually, if you look at [the list](https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L19426), 39 out of 119 (33%) don't have their own type.  you learn something every day! :)

Reply:
fixed in https://github.com/elastic/eui/pull/2835/commits/b85919b109e32233822226cbb3330d6d5ed93483